contrib/debian/copyright

```diff
@@ -1235,3 +1244,28 @@ License: GPLv3-or-later-with-Autoconf-exception
 Comment:
  The "special exception" is extended to the versions of files covered by
  this license that are distributed with this software.
+
+License: X11
```

Comment:
This is quite nitpicky, but the X11 license specifically mentions the X Consortium as the party that the no-advertising clause applies to.

```suggestion
License: Expat-with-advertising-clause
```